configure.ac

```diff
@@ -386,22 +385,16 @@ if test "${shapelib_desired}" = "yes" ; then
     LIBS="$LIBS -lshp"
     AC_CHECK_LIB(shp,DBFGetFieldIndex,
       AC_DEFINE(HAVE_DBFGETFIELDINDEX, , Define to 1 if your `shp' library has DBFGetFieldIndex. ) )
-    AC_DEFINE(HAVE_LIBSHP, , Define to 1 if you have the `shp' library (-lshp). )
```

Comment:
HEEEEEERE's your problem.  You should not remove this AC_DEFINE, because it is necessary to enable compilation of various ifdef'd code in dbfawk.c and map_shp.c.